packages/core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -289,7 +289,8 @@ const Dropdown: VibeComponent<DropdownComponentProps, HTMLElement> & {
     );
     const onChange = (option: DropdownOption | DropdownOption[], meta: ActionMeta<DropdownOption>) => {
       if (customOnChange) {
-        customOnChange(option, meta);
+        const newValue = multi ? (option.length > 0 ? option : null) : option;
```

Comment:
if multi and array is empty return null, to mimic current behavior, and ignore the behavior of the new react-select's api for isMulti